plone/app/widgets/static/requirejs-unset.js

```diff
@@ -1,4 +1,4 @@
-var _old_define = define;
```

Comment:
did you get an error here?
I wonder, because I'd have expected, if _old_define isn't defined, than define is just set to undefined.